### PR TITLE
fix(e2e): Document test is executed in Manifest based environments

### PR DESCRIPTION
**Pull request by SimonBaeumer**
## Description
Add doc that test is only executed in Manifest based environments

### CODE REVIEW

File: qa-tests-backend/src/test/groovy/CertRotationTest.groovy [hunk 1]
```diff
@@ -176,6 +176,7 @@ class CertRotationTest extends BaseSpecification {
         return true
     }
 
+    // This test is skipped in Operator and Helm, upgrader only works in Manifest based installations
     def "Test sensor cert rotation with upgrader succeeds for non-Helm clusters"() {
```
Comment (tommartensen) [inline qa-tests-backend/src/test/groovy/CertRotationTest.groovy:180]: Would it make sense to rephrase this to 
```suggestion
    def "Test sensor cert rotation with upgrader succeeds for manifest based installed clusters"() {
```

(if yes, also change line 181)

Comment (SimonBaeumer) [reply qa-tests-backend/src/test/groovy/CertRotationTest.groovy]: done

**Review by tommartensen — APPROVED** [review]
LGTM
